SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebuggerEditor.cpp

```diff
@@ -0,0 +1,106 @@
+#include "Utils/SpatialDebuggerEditor.h"
+
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include "LoadBalancing/LayeredLBStrategy.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/InspectionColors.h"
+#include "Debug/DebugDrawService.h"
+#include "Editor.h"
+#include "LevelEditor.h"
+#include "Modules/ModuleManager.h"
+#include "Engine/World.h"
+#include "GameFramework/WorldSettings.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+
+using namespace SpatialGDK;
+
+ASpatialDebuggerEditor::ASpatialDebuggerEditor(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	PrimaryActorTick.bCanEverTick = false;
+
+	bReplicates = false;
+}
+
+void ASpatialDebuggerEditor::Destroyed()
+{
+	DestroyWorkerRegions();
+}
+
+void ASpatialDebuggerEditor::ShowWorkerRegions(bool bEnabled)
```

Comment:
Maybe rename to something like `ToggleWorkerRegionVisibility(bool)` which seems to reflect that we use the method to both show and hide?

Reply:
Sure.